js/deno.ts

```diff
@@ -13,5 +13,41 @@ export {
   lstatSync,
   writeFileSync
 } from "./os";
+export {
+  DenoError,
+  ErrNotFound,
+  ErrPermissionDenied,
+  ErrConnectionRefused,
+  ErrConnectionReset,
+  ErrConnectionAborted,
+  ErrNotConnected,
+  ErrAddrInUse,
+  ErrAddrNotAvailable,
+  ErrBrokenPipe,
+  ErrAlreadyExists,
+  ErrWouldBlock,
+  ErrInvalidInput,
+  ErrInvalidData,
+  ErrTimedOut,
+  ErrInterrupted,
+  ErrWriteZero,
+  ErrOther,
+  ErrUnexpectedEof,
+  ErrEmptyHost,
+  ErrIdnaError,
+  ErrInvalidPort,
+  ErrInvalidIpv4Address,
+  ErrInvalidIpv6Address,
+  ErrInvalidDomainCharacter,
+  ErrRelativeUrlWithoutBase,
+  ErrRelativeUrlWithCannotBeABaseBase,
+  ErrSetHostOnCannotBeABaseUrl,
+  ErrOverflow,
+  ErrHttpUser,
+  ErrHttpClosed,
+  ErrHttpCanceled,
+  ErrHttpParse,
+  ErrHttpOther
+} from "./errors";
```

Comment:
Is it not possible to do `export * from "./errors"` ? I never understood why that doesn't work?

Reply:
No, it's not possible :(

Reply:
Blame TC39... 😁

Reply:
Why don't we just drop the Err prefix from those class names, and then do `export * as error from "./errors"` ?

Reply:
I want to maintain a flat namespace.

Reply:
> I want to maintain a flat namespace.

What was the reason for that again?

Reply:
@piscisaureus I want to add something to the roadmap explaining in detail the flat namespace. But let me outline it here:
1. It's hard to remember what modules things come from - users often just remember the function name. For example in Node and python we have functionality split between many modules: sys, os, process, fs, where is what? Was it "os.path.exists" or "os.exists" ? Was it "process.chdir()" or "os.chdir()"?
2. A flat namespace forces us to be thoughtful and descriptive in our names. I consider it sloppy design to have two "TempDir()" functions as was done in Golang - what does os.TempDir() do and what does ioutil.TempDir() do?
3. It makes our documentation simple: there can be an alphabetical list of symbols. No need to search thru multiple subpages. 
4. Deno is a single software package, and the symbols we export should be exposed as such.
5. Deno is very much like an operating system - syscalls and errnos are in a single namespace.